pydis_site/apps/api/viewsets/bot/offensive_message.py

```diff
@@ -0,0 +1,66 @@
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework.filters import SearchFilter
+from rest_framework.mixins import (
+    CreateModelMixin,
+    DestroyModelMixin,
+    ListModelMixin,
+    UpdateModelMixin
+)
+from rest_framework.viewsets import GenericViewSet
+
+from pydis_site.apps.api.models.bot.offensive_message import OffensiveMessage
+from pydis_site.apps.api.serializers import OffensiveMessageSerializer
+
+
+class OffensiveMessageViewSet(
```

Comment:
I'd like to see tests for this model that guarantee the API we want for it. This means that if we ever change anything in the settings later, the tests will reveal if it had any undesirable side-effects on the endpoint.